pom.xml

```diff
@@ -8,7 +8,7 @@
 
   <groupId>com.echobox</groupId>
   <artifactId>ebx-linkedin-sdk</artifactId>
-  <version>1.0.2</version>
+  <version>1.0.3</version>
```

Comment:
No need to bump the version I think it's fine to release it as part of the current snapshot

Reply:
fixed